source/_docs/modules.md

```diff
@@ -39,6 +39,71 @@ This module provides general methods your site needs to access aspects of the in
 
 Note: Currently, there is no API module available for Drupal 8.
 
+Issue: For sites with Filefield Paths module enabled and a lot of image styles, some transactions may hit some performance issues as image paths are flushed due to its volume. 
+
+Workaround: Add the image path files clearing execution using the Drupal Queue API and be executed during cron run
+
+```
```

Comment:
@eabquina Where is this (what file), and it is an addition or a modification?

Reply:
@alexfornuto  This is for the file in the Pantheon API module

https://github.com/pantheon-systems/drops-7/blob/default/modules/pantheon/pantheon_api/pantheon_api.module